test/integration/ackermann_steering_system.cc

```diff
@@ -637,6 +637,158 @@ TEST_F(AckermannSteeringOnlyTest,
   pub.Publish(msg);
 }
 
+/////////////////////////////////////////////////
+TEST_P(AckermannSteeringTest,
+    GZ_UTILS_TEST_DISABLED_ON_WIN32(
+      AngularSpeedLimiters))
+{
+  const std::string &_sdfFile = common::joinPaths(std::string(PROJECT_SOURCE_PATH), "/test/worlds/ackermann_steering_with_angular.sdf");
```

Comment:
use `common::joinPaths`

Reply:
Sorry, what is the change? Because it is already using `common::joinPaths()`

Reply:
```
  const std::string &_sdfFile = common::joinPaths(std::string(PROJECT_SOURCE_PATH, "test", "worlds", "ackermann_steering_with_angular.sdf"));
```